server/controllers/routes/user/post.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable camelcase */
+const { insertUser } = require('./../../../database/queries/insertUser');
+const { userPostSchema } = require('./../../utils/validationSchemes');
+const { hashPassword } = require('./../../utils/helper');
+
+exports.post = (req, res, next) => {
+  const userInfo = { ...req.body.user };
+  userInfo.avatar = 'avatar.png';
+  userPostSchema
+    .validate(userInfo)
+    .then((valid) => {
+      if (valid) return hashPassword(userInfo.password, 10);
```

Comment:
your helper function doesn't talk the number of rounds as an argument check utils/helper.js
```suggestion
      if (valid) return hashPassword(userInfo.password);
```